pom.xml

```diff
@@ -145,6 +166,16 @@ SOFTWARE.
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <version>${maven-surefire-plugin.version}</version>
+        <configuration>
```

Comment:
I'm still wondering which approach is better, using `yml` configurations with profiles or this, wdyt?

Reply:
@l3r8yJ we can try `yml`, but in bot, we already using XML config properties

Reply:
@h1alexbel yes, I think to use `yml` in the bot as well

Reply:
@l3r8yJ can we create a ticket for now? Since I don't know how to do it yml now

Reply:
@h1alexbel yeah, go ahead

Reply:
@l3r8yJ created, https://github.com/Blamer-io/hub/issues/37 and https://github.com/Blamer-io/bot/issues/59